app/models/target.rb

```diff
@@ -10,4 +11,10 @@ def matches
                     " ll_to_earth(latitude, longitude)) <= (#{length} + length)"
     Target.shared_topic(self).where(radius_query).to_a
   end
+
+  private
+
+  def validate_targets_limit
+    errors.add(:user, "limit reached") if user && user.targets.count >= 10
```

Comment:
Use single quotes whenever possible.

Reply:
When would `user` be nil? Is it possible?

Reply:
Set the max targets in an ENV var or a constant easy to locate.

Reply:
`user` is nil in the shoulda test when validating it's presence.

Reply:
Ah, I see. Actually in any case where the target does not have a user it will break because all validations run regardless of if a previous one already failed. 

This is fine then I guess. Ignore the comment.